element_array_ephys/ephys_organoids.py

```diff
@@ -138,46 +131,45 @@ class AcquisitionSoftware(dj.Lookup):
     contents = zip(["SpikeGLX", "Open Ephys", "Intan"])
 
 
+@schema
+class Port(dj.Lookup):
+    definition = """  # Port ID of the Intan acquisition system
+    port_id     : char(2)
+    """
+    contents = zip(["A", "B", "C", "D"])
+
+
 @schema
 class EphysRawFile(dj.Manual):
-    definition = f""" # Catalog of raw ephys files
-    file_path         : varchar(512) # path to the file on the external store
+    definition = """ # Catalog of all raw ephys files
+    file_path         : varchar(512) # path to the file on the external store relative to the root directory
     ---
-    -> [nullable] Subject
+    -> [nullable] culture.Experiment
     -> AcquisitionSoftware
-    file_time         : datetime     #  date and time of the file acquisition
+    file_time         : datetime #  date and time of the file acquisition
     parent_folder     : varchar(128) #  parent folder containing the file
     filename_prefix   : varchar(64)  #  filename prefix, if any, excluding the datetime information
-    file              : filepath@{FILE_STORE}  
+    file              : filepath@data-root
     """
 
 
 @schema
 class EphysSession(dj.Manual):
-    """User defined ephys session for downstream analysis.
-
-    Attributes:
-        Subject (foreign key): Subject primary key.
-        insertion_number (tinyint, unsigned): Unique insertion number for each probe and electrode configuration for a given subject.
-        start_time (datetime): Start date and time of session used for analysis.
-        end_time (datetime): End date and time of session used for analysis.
-        session_type (enum): Downstream analysis method to be performed ("lfp", "spike_sorting", "both").
-    """
-
-    definition = """
-    -> Subject
+    definition = """ # User defined ephys session for downstream analysis.
+    -> culture.Experiment
     insertion_number            : tinyint unsigned
     start_time                  : datetime
     end_time                    : datetime
     ---
     session_type                : enum("lfp", "spike_sorting", "both") # analysis method
+    -> [nullable] culture.Drug
+    drug_concentration=null     : float # concentration in uM
```

Comment:
Actually, this is downstream from that. Since `culture.Experiment` serves as `subject.Subject` for this project, this will correspond to `session.Session`